rstudio/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN apt-get update -y &&  \
          sqlite3 \
          libopencv-dev \
          libsqliteodbc \
-         python-matplotlib  && \
+         python-matplotlib  \
+         python-gdal && \
     easy_install pip && \
     pip install \
```

Comment:
can we use a requirements.txt file that has specific versions listed of the packages. That will help with issues where it might pick just the latest release and result in unexpected results (including build errors).

Reply:
@robkooper Do you mean the requirements.txt that is maintained in the terrautils repository?

Reply:
after the `pip install` do a `pip freeze > requirements.txt` this should be it. I'd recommend to clean it up a little to just list the specific ones that are installed using pip install. That being said it is just a comment and should not prevent this pull request from being merged.